src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -342,9 +343,19 @@ const PIAFormPage = () => {
 
   const fetchAndUpdatePia = async (piaId: string) => {
     const updatedPia = (
-      await HttpRequest.get<IPiaFormResponse>(
-        API_ROUTES.GET_PIA_INTAKE.replace(':id', `${piaId}`),
-      )
+      search && search.includes('invite')
```

Comment:
wondering removing these lines may impact the pia updates.. Have we validated that? Unfortunately we don't have Tests to validate it. But if you have, happy to approve it

Reply:
Tested on the surface - and looks like it works fine.  And assuming you'd have already tested yourself, approving.